fbgemm_gpu/include/fbgemm_gpu/fbgemm_cuda_utils.cuh

```diff
@@ -1461,8 +1476,21 @@ DEVICE_INLINE half16 to_half16(float_16 v) {
   return t;
 }
 
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
Same here

Reply:
It's addressed.

Reply:
Thanks!  Re-running the internal builds and checks at the moment to see if we're missing anything else